compose-dev.yaml

```diff
@@ -0,0 +1,17 @@
+# *
```

Comment:
This is a new file. I suspect this may have been added accidentally. If so, can it be removed? If not, can you help me understand its use and why it is in this directory?